andela/Lit-Kigali1-Frontend
# #163519111 Add Notifications

#### What does this PR do?
- Add capability of giving users notifications

#### Description of Task to be completed?
- Add notification Icon
- Add capability of giving notifications to a logged in user.

#### How should this be manually tested?

- Pull this branch

- run npm install

- run npm test

- run npm start

- Login and you will be able to see the notification icon on the navigation bar.

#### Any background context you want to provide?
- You must be authenticated to receive notifications

#### What are the relevant pivotal tracker stories?
#163519111

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/22234193/57516942-090f8e80-7316-11e9-90c5-9251f560f7bd.png)

![image](https://user-images.githubusercontent.com/22234193/57516961-13318d00-7316-11e9-9637-390cf9b2818e.png)


#### Questions:
